Controllers/General Roles Controllers/patientController.js

```
@@ -0,0 +1,53 @@
+            error: "PatientController- Get Patient Diagnosis: Unauthorized",
+          });
+        }
+      }
+
+      const public_data = await PatientService.findPatient(_id);
+      res.status(200).json(public_data.diagnosis);
+    } catch (fetchPatientDiagnosisError) {
+      res.status(500).json({ error: fetchPatientDiagnosisError.message });
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)